### ISSUE TITLE
SyntaxError: Cannot use 'import.meta' outside a module

### ISSUE THREAD

**Issue by BenjaminIrwin** [danstepanov/create-expo-stack#56]
I follow the steps to generate the code, with successful library installation. I then follow the steps suggested (Firebase project setup and putting the creds + relevant info in the .env file). Unfortunately I'm met with this error:

```
const defaultConfig = getDefaultConfig(import.meta.dir);
                                              ^^^^

SyntaxError: Cannot use 'import.meta' outside a module
```

This happens when I try to run the code for the first time with `npm run ios`.

**Comment by danstepanov** [comment]
Thanks for flagging this! Can you please post the full output of you running `npx create-expo-stack`? That will help us debug the issue quickly.

**Comment by danstepanov** [comment]
I was able to recreate the issue following the steps for the following stack:

```
 ~ npx create-expo-stack@latest
                          _
   ___  _ __  ___   __ _ | |_  ___
  / __|| '__|/ _ \ / _` || __|/ _ \
 | (__ | |  |  __/| (_| || |_|  __/
  \___||_|   \___| \__,_| \__|\___|
   ___ __  __ _ __    ___
  / _ \\ \/ /| '_ \  / _ \
 |  __/ >  < | |_) || (_) |
  \___|/_/\_\| .__/  \___/
       _     |_|        _
  ___ | |_  __ _   ___ | | __
 / __|| __|/ _` | / __|| |/ /
 \__ \| |_| (_| || (__ |   <
 |___/ \__|\__,_| \___||_|\_\

✔ What do you want to name your project? (my-expo-app) ·
✔ Would you like to use TypeScript with this project? (Y/n) · true
Good call, now using TypeScript! 🚀
✔ What would you like to use for Navigation? · React Navigation
✔ What type of navigation would you like to use? · Stack
Great, we'll use React Navigation!
✔ What would you like to use for styling? · Tamagui
You'll be styling with ease using Tamagui.
✔ What would you like to use for authentication? · Firebase
You'll be using Firebase for authentication.

Your project configuration:
{
  projectName: 'my-expo-app',
  packages: [
    {
      name: 'react-navigation',
      type: 'navigation',
      options: { type: 'stack' }
    },
    { name: 'tamagui', type: 'styling' },
    { name: 'firebase', type: 'authentication' }
  ],
  flags: { noGit: undefined, noInstall: undefined, importAlias: '~/' }
}

To recreate this project, run:
  npx create-expo-stack@latest my-expo-app --react-navigation --tamagui --firebase --npm

Initializing your project...

Copying base assets...

Installing dependencies using npm...


Cleaning up your project...


> my-expo-app@1.0.0 format
> eslint '**/*.{js,jsx,ts,tsx}' --fix && prettier '**/*.{js,jsx,ts,tsx,json}' --write

app.json 19ms
App.tsx 77ms (unchanged)
babel.config.js 5ms (unchanged)
metro.config.js 2ms (unchanged)
package-lock.json 81ms (unchanged)
package.json 0ms
src/navigation/index.tsx 7ms (unchanged)
src/screens/details.tsx 3ms (unchanged)
src/screens/overview.tsx 5ms (unchanged)
tamagui.config.ts 10ms (unchanged)
tsconfig.json 0ms
utils/firebase.ts 2ms (unchanged)

Initializing git...

Success! 🎉 Now, here's what's next:

Head over to https://console.firebase.google.com/ to create a new Firebase project.

Get the API key and other unique identifiers:
1. Register a web app in your Firebase project:
https://firebase.google.com/docs/web/setup#register-app
2. Find your API key and other identifiers.
3. Copy these keys and paste them into your .env file.
4. Optionally, follow one of these guides to get started with Firebase:
https://docs.expo.dev/guides/using-firebase/#next-steps

Once you're done, run the following to get started:

cd my-expo-app
npm run ios

If you frequently use create expo stack, please consider sponsoring the project ❤️
https://github.com/sponsors/danstepanov
```

The error occurs in https://github.com/danstepanov/create-expo-stack/blob/beta/cli/src/templates/packages/firebase/metro.config.js

```
my-expo-app git:(main) npm run ios

> my-expo-app@1.0.0 ios
> expo start --ios

Starting project at /Users/dan/my-expo-app
env: load .env
env: export EXPO_PUBLIC_FIREBASE_API_KEY EXPO_PUBLIC_FIREBASE_AUTH_DOMAIN EXPO_PUBLIC_PROJECT_ID EXPO_PUBLIC_STORAGE_BUCKET EXPO_PUBLIC_MESSAGING_SENDER_ID EXPO_PUBLIC_APP_ID EXPO_PUBLIC_MEASUREMENT_ID
SyntaxError: Cannot use 'import.meta' outside a module
/Users/dan/my-expo-app/metro.config.js:3
const defaultConfig = getDefaultConfig(import.meta.dir);
                                              ^^^^

SyntaxError: Cannot use 'import.meta' outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at module.exports (/Users/dan/my-expo-app/node_modules/cosmiconfig/node_modules/import-fresh/index.js:28:9)
    at loadJs (/Users/dan/my-expo-app/node_modules/cosmiconfig/dist/loaders.js:9:18)
    ```

**Comment by danstepanov** [comment]
This should be resolved in v2.2.13, try it using `npx create-expo-stack@latest`. If you prefer to keep your current project, you just need to go into your `metro.config.js` and change `import.meta.dir` to `__dirname`.

**Comment by BenjaminIrwin** [comment]
Thank you!
